src/renderer/components/+namespaces/namespace-select-filter.tsx

```diff
@@ -49,14 +61,22 @@ const Placeholder = observer((props: PlaceholderProps<any, boolean>) => {
   };
 
   return (
-    <components.Placeholder {...props}>
+    <components.Placeholder {...props} data-testid="namespace-select-filter-placeholder">
```

Comment:
_nit:_ Noticed in some places we're using `data-test-id` too (probably better to stick to single definition)

Reply:
It's better to stick with `data-testid` since we use it a lot more often. (102 search results vs 6 `data-test-id`.